### ISSUE TITLE
Helm Status missing notes

### ISSUE THREAD

**Issue by tgeek77** [IBM/kui#1507]
When doing `helm status <releasename>`, the deployment nodes should be displayed.

For example, when deploying the Helm chart fo SUSE Cloud Application Platform, your Cloud Foundry endpoint is displayed in the status notes:

```
NOTES:
    Welcome to your new deployment of SCF.

    The endpoint for use by the `cf` client is
        https://api.cap111.example.com

    To target this endpoint run
        cf api --skip-ssl-validation https://api.cap111.example.com

    Please remember, it may take some time for everything to come online.

    You can use
        kubectl get pods --namespace scf

    to spot-check if everything is up and running, or
        watch -c 'kubectl get pods --namespace scf'

    to monitor continuously.

    The online documentation (release notes, deployment guide) can be found at
        https://www.suse.com/documentation/cloud-application-platform-1
```
However if I do helm status in Kui, I don't see the status notes.

**Comment by starpit** [comment]
thanks for the bug report! we will look into this.

**Comment by starpit** [comment]
@ruanxianzhi will be investigating this today!

**Comment by starpit** [comment]
fixed by #1623 

the fix is live in build 2412; the full version is 2.33.0-dev.**2412**.20190605221858
